browser/src/control/Control.Menubar.ts

```diff
@@ -2183,6 +2183,8 @@ class Menubar extends L.Control {
 			|| id === 'print-all-sheets'
 			|| id === 'serveraudit') {
 			app.dispatcher.dispatch(id);
+		} else if (id === ('settings-dialog')) {
+			this._map.settings.showSettingsDialog();
```

Comment:
we rather try to move "globally accessible" components to `app.something` than `map.something`, just FYI, doesn't have to be urgently moved

Reply:
also: we should put new "online commands" to dispatcher (app.dispatcher.dispatch(...))
then it can be easily reused everywhere. can be converted later

Reply:
noted